modules/tabs-v2/assets/scss/frontend.scss

```diff
@@ -49,7 +51,10 @@
 	--tabs-v2-title-shadow: none;
 
 	--tabs-v2-icon-color: var( --e-global-color-secondary, var( --tabs-v2-color-secondary-fallback ) );
-	--tabs-v2-icon-active-color: #{$white};
+	--tabs-v2-icon-color-hover: var( --tabs-v2-title-color-hover );
+	--tabs-v2-icon-color-active: #{$white};
+	--tabs-v2-icon-size: initial;
```

Comment:
The `.elementor-tab-icon` element should receive a `display: flex` property to make sure the icon is vertically centered when it is smaller than the title text.

Reply:
thanks!

updated without css variable > I don't think there is any need for a variable here

![image](https://user-images.githubusercontent.com/94352322/177297319-965eea51-cac3-4b68-bbad-1781e83e7f2d.png)

Reply:
Why do you need the display: flex both in the main title selector and the -active selector?